CMakeLists.txt

```diff
@@ -436,6 +436,9 @@ endif()
 set_source(simdjson)
 resolve_dependency(simdjson 3.1.5)
 
+set(cpr_SOURCE BUNDLED)
```

Comment:
Can we add this only if `VELOX_BUILD_TESTING=ON`?

Reply:
Sure, done.